config/utilities/upperCaseFirstCharacter.test.ts

```diff
@@ -0,0 +1,19 @@
+import {upperCaseFirstCharacter} from './upperCaseFirstCharacter'
+
+describe('Utilities: upperCaseFirstCharacter', () => {
+  it('it transforms all lowercase word', () => {
+    expect(upperCaseFirstCharacter('primer')).toStrictEqual('Primer')
+  })
+
+  it('it transforms all lowercase sentence (words with spaces)', () => {
+    expect(upperCaseFirstCharacter('primer design token')).toStrictEqual('Primer design token')
+  })
+
+  it('it transforms all uppercase word', () => {
```

Comment:
is this a more accurate test name?
```suggestion
it('it preserves casing for words that are already all uppercased', () => {
```